presentation/backend/src/presentation-backend.ts

```diff
@@ -8,14 +8,11 @@
  * @docs-group-description Core
  * Common types used for retrieving presentation data from iModels.
  */
-export { Presentation, PresentationProps, PresentationPropsDeprecated, PresentationPropsNew } from "./presentation-backend/Presentation"; // eslint-disable-line deprecation/deprecation
-export {
-  HierarchyCacheMode, HierarchyCacheConfig, HierarchyCacheConfigBase, HybridCacheConfig, MemoryHierarchyCacheConfig,
-  DiskHierarchyCacheConfig, PresentationManager, PresentationManagerProps, PresentationManagerMode,
-} from "./presentation-backend/PresentationManager";
-export { RulesetManager } from "./presentation-backend/RulesetManager";
-export { RulesetVariablesManager } from "./presentation-backend/RulesetVariablesManager";
-export { RulesetEmbedder, RulesetEmbedderProps, DuplicateRulesetHandlingStrategy } from "./presentation-backend/RulesetEmbedder";
+export * from "./presentation-backend/Presentation";
+export * from "./presentation-backend/PresentationManager";
```

Comment:
@grigasp you good with this change?

Reply:
Yes, I was going to make this change at some point anyway. Thanks.